.github/workflows/build.yml

```diff
@@ -135,7 +135,7 @@ jobs:
           exit 1
         fi
         echo "Running tests..."
-        if [[ "x${{ github.event.inputs.ignoreLowerCoverage }}" == "xtrue" ]]; then
+        if [[ "x${{ github.event.inputs.ignoreLowerCoverage }}" == "true" ]]; then
```

Comment:
Can this ever be true? (note the "x" prefix on the LHS)

Reply:
Yeah that's wrong. I incorrectly assumed that extra x on the RHS was a typo but it's supposed to be there. I'll "unfix" the fix on main.